src/descriptors/power_system_inputs.json

```diff
@@ -219,37 +219,37 @@
         {
             "unit": "MW",
             "name": "active_power",
-            "description": "Real power injection setpoint",
+            "description": "Real power StaticInjection setpoint",
```

Comment:
Looks like you have a lot of accidental name replacements for injection.  You should review all of them.